virtualmicroscope/views.py

```diff
@@ -46,4 +51,7 @@ def custom_index(request, conn=None, **kwargs):
     context['template'] = 'virtualmicroscope/start.html'
     return context
 
-omeroweb.webstart.views.custom_index = custom_index
+try:
```

Comment:
I would propose to define a `legacy` flag in the top-level import block and use it in a traditional `if/else` block here